packages/flutter/test/widgets/semantics_tester.dart

```diff
@@ -340,6 +340,54 @@ class SemanticsTester {
     visit(tester.binding.pipelineOwner.semanticsOwner.rootSemanticsNode);
     return result;
   }
+
+  /// Generates an expression that creates a [TestSemantics] reflecting the
+  /// current tree of [SemanticsNode]s.
+  ///
+  /// Use this method to generate code for unit tests.
```

Comment:
Can you document this a little more? Why would I want to use this for unit tests?

Reply:
I feel sightly uncomfortable about this addition. Will it incentive people to just regenerate the test when it fails and accept whatever its output is without investigating if there is an actual breakage?

Reply:
I hope that it will work similar to screenshot testing. The first time you check in a test it's a big chunk of code (although we should definitely prevent people from submitting huge piles of semantics trees). However, subsequent changes should be incremental. For example, if you change the label on the "Next month" button, it will be a one-line change in the test. Or if you add one more `SemanticsNode` to the tree, it will be one more `TestSemantics` expression. Both should be easy to review. A huge change to the semantics tree would trigger big code changes in the test and alert the reviewer (and author) that a more thorough verification of the change needs to be done.

I will add this explanation to the dartdoc.

Reply:
Sounds good.